components/nexus-repository/src/main/java/org/sonatype/nexus/repository/proxy/ProxyFacetSupport.java

```diff
@@ -212,7 +212,7 @@ protected Content fetch(String url, Context context, Content stale) throws IOExc
 
       Payload payload = new HttpEntityPayload(response, entity);
       final Content result = new Content(payload);
-      result.getAttributes().set(Content.CONTENT_LAST_MODIFIED, extractLastModified(response.getLastHeader(HttpHeaders.LAST_MODIFIED)));
+      result.getAttributes().set(Content.CONTENT_LAST_MODIFIED, extractLastModified(request.getURI(), response.getLastHeader(HttpHeaders.LAST_MODIFIED)));
```

Comment:
Perhaps just pass in the `request` to avoid the cost of calling `getURI()` when there is no problem?  Also is this information that useful, given that the request is already logged (albeit at debug) and any issue with parsing should presumably be easy to recreate?